extensions/s3/src/main/java/io/deephaven/extensions/s3/Credentials.java

```diff
@@ -8,8 +8,7 @@ public interface Credentials {
     /**
      * The default credentials.
      *
-     * @see <a href="https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials-chain.html">Default
-     *      credentials provider chain</a>
+     * @see DefaultCredentials
```

Comment:
DefaultCredentials is not public; should describe the behavior here.

Reply:
Will do